windowing.py

```diff
@@ -205,3 +205,84 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def place_data(np_sample, data_key, data_label, dictionary, positive, window_size, stride):
+    """
+        Description: windows the np_sample (np array) with desired specifications and then
+                     places the windows into dictionary["data_key"] and creates a
+                     corresponding number of labels to dictionary["data_label"]
+
+        Args:
+            np_sample -- np array
+            data_key -- String (key within dictionary to place windows: dictionary[data_key])
+            data_label -- String (key within dictionato place label: dictionary[data_label])
+            dictionary -- dict (dictionary where windows and labels will be added to)
+            positive -- boolean (binary label of windows)
```

Comment:
Would using an int/enum with predefined class ids make the function more general?